source/common/http/headers.h

```diff
@@ -313,6 +316,11 @@ class HeaderValues {
     const std::string Nosniff{"nosniff"};
   } XContentTypeOptionValues;
 
+  struct {
+    const std::string Window{"window="};
+    const std::string Name{"name="};
```

Comment:
`name` lets it be pretty broad which is maybe a good thing (I see you used it for "first" and "second"  in the tests which I don't fully understand) but in most cases here wouldn't it be just the unit?

Reply:
I guess I should rename the const. These are the keys for the policy definition as defined in https://tools.ietf.org/id/draft-polli-ratelimit-headers-00.html#rfc.section.3.1 - the quota-policy can contain arbitrary keys describing it. And in our protocol we have names for the descriptors, so I thought why not use it?